src/main/java/calculator/controller/CalculatorController.java

```diff
@@ -0,0 +1,33 @@
+package calculator.controller;
+
+import calculator.model.CalculatorService;
+import calculator.model.StringService;
+import calculator.view.InputView;
+import calculator.view.OutputView;
+import java.util.List;
+
+public class CalculatorController {
+
+    InputView inputView;
+    OutputView outputView;
+    CalculatorService calculatorService;
+    StringService stringService;
+
+    public CalculatorController(InputView inputView, OutputView outputView, CalculatorService calculatorService,
+                                StringService stringService) {
+        this.inputView = inputView;
+        this.outputView = outputView;
+        this.calculatorService = calculatorService;
+        this.stringService = stringService;
+    }
+
+    public void calculate() {
+        String inputString = inputView.input();
+
+        /* Controller가 실행 흐름 담당 */
+        List<Integer> integerList = stringService.parse(inputString);
+        Integer result = calculatorService.sum(integerList);
```

Comment:
자료형을 그대로 작성하는 것보다는 변수명에 의미가 담겨있으면 더 알아보기 편하겠네요! 👍 

```suggestion
        /* Controller가 실행 흐름 담당 */
        List<Integer> numbers = stringService.parse(inputString);
        Integer result = calculatorService.sum(numbers);
```

Reply:
좀 더 변수명에 의미를 담아보겠습니다! 피드백 감사합니다 👍